Sources/Tracing/TracingTime.swift

```diff
@@ -21,26 +21,17 @@ import Darwin
 @_exported import Instrumentation
 @_exported import InstrumentationBaggage
 
-public protocol SwiftDistributedTracingDurationProtocol: Comparable, AdditiveArithmetic, Sendable {
-    static func / (_ lhs: Self, _ rhs: Int) -> Self
-    static func /= (_ lhs: inout Self, _ rhs: Int)
-    static func * (_ lhs: Self, _ rhs: Int) -> Self
-    static func *= (_ lhs: inout Self, _ rhs: Int)
-
-    static func / (_ lhs: Self, _ rhs: Self) -> Double
-}
-
-extension SwiftDistributedTracingDurationProtocol {
-    public static func /= (_ lhs: inout Self, _ rhs: Int) {
-        lhs = lhs / rhs
-    }
+public protocol TracerInstantProtocol: Comparable, Hashable, Sendable {
```

Comment:
This is one of the two last places where we have the `Protocol` suffix. Do we want to get rid of it entirely?

Reply:
Yeah I think we can :+1:

I'll do this in this PR actually. Implementations tend to be called Timestamp or something like that after all